test/e2e_node/jenkins/jenkins-docker-validation.properties

```diff
@@ -0,0 +1,15 @@
+GCI_IMAGE_PROJECT=container-vm-image-staging
+GCI_IMAGE_FAMILY=gci-preview-test
+GCI_IMAGE=$(gcloud compute images describe-from-family ${GCI_IMAGE_FAMILY} --project=${GCI_IMAGE_PROJECT} --format="value(name)")
+DOCKER_VERSION=$(curl -fsSL --retry 3 https://api.github.com/repos/docker/docker/releases | tac | tac | grep -m 1 "\"tag_name\"\:" | grep -Eo "[0-9\.rc-]+")
```

Comment:
@wonderfly Agree. We could download it ourselves in this script, but that will introduce another dependency and make the code a little more complex. :)